### DISCUSSION TITLE
Weird rendering

### DISCUSSION THREAD

**Discussion by 0xDmtri** [NvChad/NvChad#1914] [Q&A] [answered]
For some reason I am having weird rendering. Bottom left corner and the question mark in the theme search bar. I am using Hack Nerd Font (the usual one, not mono). And I am running Neovim 9.

<img width="825" alt="Screenshot 2023-04-12 at 1 33 21 AM" src="https://user-images.githubusercontent.com/33504992/231317360-683dbe5b-214d-4abb-8170-6ce8a1fafb43.png">

**Comment by siduck** [comment, marked as answer]
@0xDmtri use latest nerd font and dont use iterm2.  try kitty or alacritty or just override telescope config and change icon
